design/open/repository-operations-error-handling.md

```diff
@@ -1,13 +1,13 @@
 # Respository Operations Error Handling
```

Comment:
I like the delete repo flow, I think it covers everything.What I like less is the introduction of new terminology. I would prefer we drop the 'disposeRepository'

Reply:
`disposeRepository` was an intermediate idea, I was playing with and is no longer a part of this document. I forgot to remove the diagram file. Will be deleted

Reply:
Removed